htdocs/projet/tasks/task.php

```diff
@@ -484,55 +484,60 @@
 			dol_fiche_end();
 		}
 
-
-		if ($action != 'edit')
+		$parameters = array();
```

Comment:
No, the additionnal button must be with the other button like in the other card page